pageserver/src/walreceiver.rs

```diff
@@ -132,7 +136,23 @@ impl<E: Clone> TaskHandle<E> {
                 TaskEvent::End(match self.join_handle.as_mut() {
                     Some(jh) => {
                         if !jh.is_finished() {
-                            warn!("sender is dropped while join handle is still alive");
+                            // Barring any implementation errors in this module, we can
+                            // only arrive here while the task that executes the future
+                            // passed to `Self::spawn()` is still execution. Cf the comment
```

Comment:
Cf? Is it meant to be check or I miss some abbreviation?

Reply:
https://en.wikipedia.org/wiki/Cf.

Reply:
Thanks!